src/main/java/org/eclipse/cargotracker/interfaces/booking/sse/RealtimeCargoTrackingService.java

```diff
@@ -29,19 +28,18 @@ public class RealtimeCargoTrackingService {
   @Inject private CargoRepository cargoRepository;
 
   @Context private Sse sse;
-  private SseBroadcaster broadcaster;
 
-  @PostConstruct
-  public void init() {
-    broadcaster = sse.newBroadcaster();
-    logger.log(Level.FINEST, "SSE broadcaster created.");
-  }
+  private SseBroadcaster broadcaster;
 
   @GET
   @Produces(MediaType.SERVER_SENT_EVENTS)
-  public void tracking(@Context SseEventSink eventSink) {
+  public void tracking(@Context SseEventSink eventSink, @Context Sse sse) {
```

Comment:
We already have an SSE instance injected. Why is this one needed also?

Reply:
You're right, this was added as a potential fix for the SSE broadcaster issue but ultimately made no difference. Will be removed in a future commit.